scripts/biomedical/VirtualMetabolicHuman/format_reaction.py

```diff
@@ -0,0 +1,221 @@
+import pandas as pd
+import sys
+import os
+import numpy as np
+import re
+
+# Convert string to list 
+def Convert(string):
+    list1=[]
+    list1[:0]=string
+    return list1
+
+
+#is_subset function for checking if the small string (metabolite abbreviation) is an exact match with a substring 
+#of the larger string (reaction abbreviation), keeping the order in place and retaining the starts with condition 
+def is_subset(list_long,list_short):
+    short_length = len(list_short)
+    subset_list = []
+    for i in range(len(list_long)-short_length+1):
+        subset_list.append(list_long[i:i+short_length])
+    for j in range(len(subset_list)):
+        if(subset_list[j] == list_short):
+            return True
+        else:
+            return False
+
+#check nan           
+def isNaN(num):
+    return num != num
+
+def main():
+    
+    file_input = sys.argv[1]
+    file_output = sys.argv[2]
+    file_gemid = sys.argv[3]
+    file_metabolite = sys.argv[4]
+
```

Comment:
comment what these args are.  its good practice to do some basic sanity check that they are all provided.

Reply:
Added a doctoring explaining the same !

Reply:
*docstring